packages/react-vapor/src/components/dropdownSearch/MultiSelectDropdownSearch/tests/DraggableSelectedOption.spec.tsx

```diff
@@ -44,6 +45,20 @@ describe('DraggableSelectedOption', () => {
             expect(selectedOption.find('.selected-option-value').first().text()).toBe(label);
         });
 
+        it('should render a custom value for the tooltip in the selectedOption', () => {
+            const customValue: string = 'Bananas';
+            mountOption({selectedTooltip: {title: customValue}});
+
+            expect(selectedOption.find(Tooltip).prop('title')).toBe(customValue);
+        });
+
+        it('should render a custom value instead of the label prop in the selectedOption', () => {
+            const customValue: string = 'Bananas';
+            mountOption({label: 'helloworld', selectedTooltip: {title: customValue}});
+
+            expect(selectedOption.find(Tooltip).prop('title')).toBe(customValue);
+        });
+

```

Comment:
I think you should add a test in the case where no selectedTooltip is provided (so you go pass through the ?? operator and it's the props.label that is used instead) ? i'm not familiar with that component though

Reply:
Thanks, after looking at the test files I found that lines 41 to 46 has covered this test case scenario where no selectedTooltip is provided and only label will be shown.